data/tests/test_merge_requests.py

```diff
@@ -0,0 +1,12 @@
+import requests_mock
+
+from django.test import TestCase
+
+from data.merge_requests import fetch_mrs
+
+
+class FetchMergeRequestTest(TestCase):
+
+    def test_fetch_mrs(self):
+        with requests_mock.Mocker():
+            fetch_mrs('GitHub')
```

Comment:
this is wrong , isnt it?  the hoster should be `github` or `gitlab` ?

that is what is in https://github.com/sks444/webservices.coala.io/tree/master/mrs , and IIRC that is what the real webservices is looking for.

Is this intentionally invalid?

If so, dont hide it with a legitimate looking string.

Put an obviously incorrect string.  e.g. `'not_valid_hoster'`

Reply:
Im told it does `.lower()` ...